cloud/db/services/organization-memberships.ts

```diff
@@ -148,9 +144,9 @@ class OrganizationMembershipBaseService extends BaseService<
  */
 export class OrganizationMembershipService extends BaseAuthenticatedService<
   PublicOrganizationMembership,
-  "organizations/:organizationId/users/:targetUserId",
+  "organizations/:organizationId/members/:memberId",
```

Comment:
note: PR description is out of date (says /members/:id). I like /members/:memberId